src/task 22/task22.js

```diff
@@ -0,0 +1,42 @@
+function buildTable(arr){
+    var Table = document.querySelector("#table");
+    var rez = arr.map(elem => (Object.keys(elem)));
+    var result = rez.flat();
+    function unique(arr) {
+        var res = [];
+      for (let str of arr) {
+    if (!res.includes(str)) {
+      res.push(str);
+    }
+  }
+        return res;
+};
```

Comment:
1. By the course content we are limited to use ES5 specs only. Arrow functions, let keyword are not the part of it.
2. forEach method is preferred to iterate through the array,
3. Try to make things shorter.